content/docs/hooks-rules.md

```diff
@@ -6,33 +6,33 @@ next: hooks-custom.html
 prev: hooks-effect.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+*Hooks* adalah sebuah fitur tambahan baru di React 16.8. Mereka membantu Anda untuk menggunakan *state* and fitur-fitur React lainnya tanpa menulis sebuah kelas.
```

Comment:
This and other Hooks-related pages have the same English sentence, but various contributors have their own version of translation. Here are PR diffs related to Hooks:

- [Diff for Using the State Hook](https://github.com/reactjs/id.reactjs.org/pull/28/files#diff-6024d880e5eef39cfbff54653ef34fb6R9) (#28):

  ```diff
  - *Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
  + *Hooks* merupakan fitur baru di React 16.8. Fitur ini memungkinkan Anda menggunakan *state* dan fitur React lainnya tanpa menuliskan sebuah kelas.
  ```

- [Diff for Using the Effect Hook](https://github.com/reactjs/id.reactjs.org/pull/38/files#diff-25b65c3d9aaad3771cfbb8ac2f5361b4R9) (#38):

  ```diff
  - *Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
  + *Hooks* adalah tambahan baru di React 16.8. *Hooks* memungkinkan Anda dapat menggunakan *state* dan fitur lain di *React* tanpa menulis *class*.
  ```

- [Diff for Hooks at a Glance (merged)](https://github.com/reactjs/id.reactjs.org/pull/88/files#diff-9bda51acb98cffd656661def0255b05bR9) (#88):

  ```diff
  - *Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
  + *Hooks* merupakan penambahan baru pada React 16.8. *Hooks* memungkinkan Anda menggunakan *state* dan fitur React lainnya tanpa membuat sebuah kelas.
  ```

I suggest we open an issue discussing whether we should sync the translations or just stick to the contributors' version.

Reply:
@grikomsn +1 for making the translations consistent. I'll go with the first one IMO.

Reply:
Which first one? If #28, #88 or a new related PR will be opened to update "Hooks at a Glance" translation.

Reply:
Ah yeah sorry @grikomsn, when I said "the first" I mean the first in that list 😅 so the one in #28.